src/Azure.Functions.Cli/Helpers/SecurityHelpers.cs

```diff
@@ -121,6 +121,10 @@ internal static async Task<X509Certificate2> CreateCertificateOpenSSL()
             return new X509Certificate2($"{certFileNames}certificate.pfx", DEFAULT_PASSWORD);
         }
 
+        // Suppressing CodeQL warning because this code is only used to validate blob content. Not used to encrypt any secrets
+        // blobClient only provides md5 string for validation. We cannot replace that unless they provide something else for validation
+        // TODO: Find out if 
+#pragma warning disable CodeQL.SM02196
```

Comment:
I don't believe this is the correct syntax to suppress CodeQL.

Reply:
I got this from co-pilot 😊 was trying to see if I could check code ql status from pr build.

Reply:
No CodeQL is not ran in PR. It is ran once every 72 hours on CI (non-PR) builds

Reply:
@soninaren curious how did you get this opaque ID - SM02196?

Reply:
fixed the syntax